### PR TITLE
add a test for the dird conference list

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: features/daily/directory.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Feature: Directory integration
+
+    Scenario: List conference rooms as a dird user
+        Given there is a user
```
Comment (fblackburn1) [inline features/daily/directory.feature]: Should we precise that the user is an authentication user and not an telephony user?
`Given there is an authentication user`

File: wazo_acceptance/steps/user.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    body = {
+        'firstname': random_string(10),
+        'username': context.username,
+        'password': context.password,
+    }
+
+    user = context.auth_client.users.new(**body)
+
+    context.add_cleanup(context.auth_client.users.delete, user['uuid'])
```
Comment (fblackburn1) [inline wazo_acceptance/steps/user.py]: If the same logic become used elsewhere, we will extract it into helper?

Comment (pc-m) [reply wazo_acceptance/steps/user.py]: Yeah I did not want to choose what to put in the user_helper module :grinning: 

I guess we will see what happens after a few steps using users

File: wazo_acceptance/steps/conference.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    given,
+    then,
+    when,
+)
+
+
+@given('there are conference rooms with infos')
+def there_are_conference_rooms(context):
+    for row in context.table:
+        body = dict(zip(row.headings, row.cells))
```
Comment (fblackburn1) [inline wazo_acceptance/steps/conference.py]: You can use row.as_dict()
https://github.com/behave/behave/blob/1cc411a82767fd63b4eff5bc9cff6d7867d45d17/behave/model.py#L1919

**Pull request merged**